src/argilla/client/sdk/workspaces/models.py

```diff
@@ -14,12 +13,33 @@
 #  limitations under the License.
 
 from datetime import datetime
+from typing import List, Optional
+from uuid import UUID
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
```

Comment:
I think that we can just import `from typing import Literal` without trying to catch the exception. `Literal` was added in Python 3.8, which is the oldest version that we support, right?

I think that I've seen this in some other places, we should update it everywhere.